modules/client/examples/05-encoders-decoders/12-apply-uninstallation.ts

```diff
@@ -0,0 +1,109 @@
+/* MARKDOWN
+---
+title: Apply Uninstallation
+---
+
+## Apply a Plugin Uninstallation
+
+Encodes the action of applying a plugin uninstallation.
+
+### Encoding
+*/
+
+import {
+  ApplyUninstallationParams,
+  Client,
+  DaoAction,
+  DecodedApplyUninstallationParams,
+} from "@aragon/sdk-client";
+import { hexToBytes } from "@aragon/sdk-common";
+import { context } from "../index";
+import { PermissionIds } from "../../dist/interfaces";
+
+// Instantiates an Aragon OSx SDK client.
+const client: Client = new Client(context);
+
+// This variable contains the values received on the prepareUninstallation() method
+const applyUninstallationParams: ApplyUninstallationParams = {
+  permissions: [{
+    condition: "0x1234567890123456789012345678901234567890",
+    operation: 1,
+    permissionId: hexToBytes(PermissionIds.EXECUTE_PERMISSION_ID),
+    where: "0x1234567890123456789012345678901234567890",
+    who: "0x2345678901234567890123456789012345678901",
+  }],
```

Comment:
If I am not mistaken, applyUninstallation should work pretty much like an installation. There is an uninstallation prepared, and the only thing missing is to apply it on the PSP. 

Not sure why these parameters are still needed here

Reply:
Reviewed with sarkawt they are needed